build/RunHelixTests.yml

```diff
@@ -0,0 +1,106 @@
+    buildOutputDir: '$(Build.SourcesDirectory)\BuildOutput'
+    buildPlatform: 'x86'
+    buildConfiguration: 'Debug'
+
+  steps:
+  - task: CmdLine@1
+    displayName: 'Display build machine environment variables'
+    inputs:
+      filename: 'set'
+
+  - task: powershell@2
+    displayName: 'Install RS5 SDK (17763)'
+    inputs:
+      targetType: filePath
+      filePath: build\Install-WindowsSdkISO.ps1
+      arguments: 17763
+
+  - task: NuGetToolInstaller@0
+    displayName: 'Use NuGet 4.9.1'
+    inputs:
+      versionSpec: 4.9.1
+        
+  - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+    displayName: 'NuGet restore MUXControls.sln'
+    inputs:
+      restoreSolution: MUXControls.sln
+      feedsToUse: config
+      nugetConfigPath: nuget.config
+
+  - task: VSBuild@1
+    displayName: 'Build solution MUXControls.sln'
+    inputs:
+      solution: MUXControls.sln
+      vsVersion: 15.0
+      platform: '$(buildPlatform)'
+      configuration: '$(buildConfiguration)'
+      msbuildArgs: '/p:AppxPackageDir="$(appxPackageDir)" /p:AppxBundle=Never /p:AppxSymbolPackageEnabled=false'
+
+  - task: powershell@2
+    displayName: 'Copy files to staging dir'
+    inputs:
+      targetType: filePath
+      filePath: build\CopyFilesToStagingDir.ps1
+      arguments: -BuildOutputDir '$(buildOutputDir)' -PublishDir '$(Build.ArtifactStagingDirectory)' -Platform '$(buildPlatform)' -Configuration '$(buildConfiguration)'
+
+  - task: PublishBuildArtifacts@1
+    displayName: 'Publish artifact: drop'
+    inputs:
+      PathtoPublish: '$(Build.ArtifactStagingDirectory)'
+      artifactName: 'drop'
+
+
+
+- job: RunTests
+  dependsOn:
+    - Build
+  pool:
+    vmImage: 'VS2017-Win2016'
+  timeoutInMinutes: 120
+  variables:
+    buildPlatform: 'x86'
+    buildConfiguration: 'Debug'
+    artifactsDir: '$(Build.SourcesDirectory)\Artifacts'
+  steps:
+
+  - task: NuGetToolInstaller@0
+    displayName: 'Use NuGet 4.9.1'
+    inputs:
+      versionSpec: 4.9.1
+
+  - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+    displayName: 'NuGet restore build/Helix/packages.config'
+    inputs:
+      restoreSolution: build/Helix/packages.config
+      feedsToUse: config
+      nugetConfigPath: nuget.config
+      restoreDirectory: packages
+
+  - task: DownloadBuildArtifacts@0 
+    inputs: 
+      artifactName: drop 
+      downloadPath: '$(artifactsDir)'
+
+  - task: powershell@2
+    displayName: 'PrepareHelixPayload.ps1'
+    inputs:
+      targetType: filePath
+      filePath: build\Helix\PrepareHelixPayload.ps1
+
+  - task: DotNetCoreCLI@2
+    displayName: 'Run tests in Helix'
+    env:
+      SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+    inputs:
+      command: custom
+      projects: build\Helix\RunTestsInHelix.proj
+      custom: msbuild
+      arguments: '/p:HelixAccessToken=$(BotAccount-dotnet-github-anon-kaonashi-bot-helix-token) /p:HelixBuild=$(Build.BuildId)'
```

Comment:
Where are you getting "BotAccount-dotnet-github-anon-kaonashi-bot-helix-token"?  Are you adding the variable group to your definition?  It's probably more sustainable to manage the variable group via yaml.

Reply:
This is from the variable group that you added to the account. I added the variable group to the Pipeline here: https://dev.azure.com/ms/microsoft-ui-xaml/_build?definitionId=22&_a=summary

What is the alternative?

Reply:
You can use variable groups in yaml like this - https://github.com/dotnet/arcade/blob/d6cab0cfb504210c9bc9aa9b7c4bfdb59edbc8f2/azure-pipelines.yml#L34

You have to define all of your variables as an array though.